be/src/common/config.h

```diff
@@ -869,7 +869,9 @@ CONF_Int64(block_cache_block_size, "1048576");  // 1MB
 CONF_Int64(block_cache_mem_size, "2147483648"); // 2GB
 CONF_Bool(block_cache_checksum_enable, "true");
 
-CONF_mInt64(l0_l1_merge_ratio, "10");
+CONF_Double(l0_l1_merge_ratio, "0.1");
```

Comment:
this may break compatibility. what's the behavior if an user originally set a value of 10, then upgrade?